contracts/CO30_Munson_It's_the_amps_that_kill_you.VTF_Korsac/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,15 @@
+<font color='#e3d310'>EMPLOYER</font>
+Name: NATO
+Location: NATO
+
+<font color='#d81717'>ENEMY FORCES</font>
+Name: SEPARATISTS
+Location: KORSCA
+
+REPORTS SHOW THAT THEY MOVED WITH SPEED USING SEVERAL MODES OF TRANSPORT. THIS INCULDES A BMP, TRUCKS, UAVS AND SEVERAL .50 CAL TECHINCALS. 
+
+<font color='#993399'>CIVILIAN CONSIDERATIONS</font>
+Name: GENERAL POPULATION
+Location: KORSCA
+
+BECAREFUL THEY'RE PRESENT IN THE AO.
```

Comment:
```suggestion
Present in the AO
```
Name and location aren't needed if there aren't any specific civilian groups (like IDAP)